eos-tech-support/eos-write-image

```diff
@@ -191,8 +191,13 @@ else
     fi
 
     if [ ! "$REMOVABLE" ]; then
-        REMOVABLE=$(lsblk --raw --nodeps --noheading --output HOTPLUG "$DEVICE")
-        if [ "$REMOVABLE" = 1 ]; then
+        if ! REMOVABLE=$(lsblk --raw --nodeps --noheading --output HOTPLUG "$DEVICE"); then
```

Comment:
I had to test that setting the variable in an `if` expression worked correctly, but it seems it does. Reading bash(1) again:
>        if list; then list; [ elif list; then list; ] ... [ else list; ] fi
>              The  if  list is executed.  If its exit status is zero, the then
>              list is executed.  Otherwise, each  elif  list  is  executed  in
>              turn,  and  if  its  exit status is zero, the corresponding then
>              list is executed and the command completes.  Otherwise, the else
>              list  is executed, if present.  The exit status is the exit sta‐
>              tus of the last command executed, or zero if no condition tested
>              true.

That does look like it's appropriate since the list is executed.